docs/manual/layout/site-structure/multi-domain-operation.en.md

```diff
@@ -26,14 +23,14 @@ RewriteRule (.*) http://www.example.org/$1 [R=301,L]
 
 Let's assume that an agency is looking after a company with several websites and wants to manage them with a central Contao installation. For this purpose, the following domains were registered:
 
-- `firma.at`
-- `firma.ch`
-- `firma.de`
+- `company.org`
```

Comment:
shouldn't we use the example.com etc. domains in the docs, since they are explicitly made for being examples? 😂

Reply:
> This domain is for use in illustrative examples in documents. You may use this domain in literature without prior coordination or asking for permission.